test-pnp.sh

```diff
@@ -1,4 +1,7 @@
 declare -a testCases=(
+  # Tests the webpack require hook
+  "progressive-web-app"  
```

Comment:
Are the failures due to this addition? Can we at least test that.

Reply:
That shouldn't affect the `Build, test, and deploy / Test Integration (5) (pull_request)` family of tests